integration_tests/src/daphne.rs

```diff
@@ -50,7 +50,7 @@ impl<'a> Daphne<'a> {
             let daphne_container = ContainerLogsDropGuard::new(
                 test_name,
                 container_client.run(runnable_image),
-                ContainerLogsSource::Docker,
+                ContainerLogsSource::Path("/logs".into()),
```

Comment:
Nice, we can probably eliminate this `ContainerLogsSource` abstraction entirely, since Daphne was the only use case for the `Docker` variant. (You don't have to do this, I'll follow up a PR :grin:).